pkg/api/api.go

```diff
@@ -33,9 +33,12 @@ type PrepareForSubmarinerInput struct {
 
 // Cloud is a potential cloud for installing Submariner on.
 type Cloud interface {
-	// PrepareForSubmariner will prepare the cloud for Submariner to operate on.
+	// Deprecated: Use OpenPorts instead, this will be removed on the 0.16 release.
```

Comment:
I think we can just remove this function now rather than adding tech debt. Users of the API can just adjust when they bump the go mod version. There's really only 2 users of this API and we control both anyway.